frontend/javascripts/oxalis/view/left-border-tabs/layer_settings_tab.tsx

```diff
@@ -242,13 +242,24 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
         onClick={() => this.deleteAnnotationLayerIfConfirmed(readableName, layer)}
         className="fas fa-trash"
         style={{
-          cursor: "pointer",
           opacity: 0.7,
```

Comment:
It might have been like this to make the icons are bit more subtle. Since you are moving these in the menu anyway, the opacity tweak can be removed, I'd assume.

Reply:
should it also be changed in the buttons outside the overflow menu?